erpnext/healthcare/doctype/patient_appointment/patient_appointment.js

```diff
@@ -332,14 +332,17 @@ let check_and_set_availability = function(frm) {
 
 		d.set_values({
 			'department': frm.doc.department,
-			'practitioner': frm.doc.practitioner,
```

Comment:
This is needed for initial form loading?

Reply:
Thanks, added it back